lib/matplotlib/axes/_axes.py

```diff
@@ -474,11 +474,22 @@ def indicate_inset(self, bounds, inset_ax=None, *, transform=None,
                 if self.yaxis.get_inverted():
                     ey = 1 - ey
                 xy_data = x + ex * width, y + ey * height
+
+                blacklist = {
```

Comment:
We already use `allowed_keys` elsewhere for this:
```suggestion
                disallowed_keys = {
```